rust-connectors/common/test-data/connectors/full-config.yaml

```diff
@@ -26,3 +26,8 @@ producer:
   compression: gzip
 consumer:
   partition: 10
```

Comment:
bump up version?

Reply:
This is test data, so the actual version value is unimportant here.